hometask2.py

```diff
@@ -0,0 +1,15 @@
+x = input("Enter a symbol:")
+if x.isdigit():
+    if int(x)%2==0:
+        print(f"{x} is even")
+    else:
+        print(f"{x} is odd")
+else:
+        print(f"{x} is a word with len of {len(x)}")
```

Comment:
It works, but to my mind, this `print` should be moved to the left to fix indentation